socorro/processor/mozilla_processor_2015.py

```diff
@@ -78,7 +68,15 @@
         "socorro.lib.transform_rules.TransformRuleSystem",
         "apply_all_rules",
         "socorro.processor.breakpad_transform_rules.JitCrashCategorizeRule, "
-        "socorro.processor.signature_utilities.SignatureJitCategory, "
+    ],
+    [
+        # generate signature now that we've done all the processing it depends on
+        'generate_signature',
+        'processor.signature',
+        'socorro.lib.transform_rules.TransformRuleSystem',
+        'apply_all_rules',
+
+        'socorro.processor.mozilla_transform_rules.SignatureGeneratorRule, '
```

Comment:
Previously, signature generating rules were split across two parts of the pipeline. We change that so there's one big signature generation part that does it all.